aszczachor/src/components/header/Header.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { AppBar, Menu, MenuItem, Tab, Tabs, Toolbar } from "@mui/material";
+import FacebookIcon from "@mui/icons-material/Facebook";
+import InstagramIcon from "@mui/icons-material/Instagram";
+import EmailIcon from "@mui/icons-material/Email";
+import { Link } from "react-router-dom";
+
+const pages = [
+  "Editorials",
+  "Videoclip",
+  "Influencers",
+  "Collections",
+  "About",
+];
+
+const Header = () => {
+  const [anchorElement, setAnchorElement] = useState(null);
```

Comment:
To jakaś copy pasta? Nie zagłębiam się w to na razie, ale trochę nie czaję tego na pierwszy rzut oka